coda-automation/terraform/buildkite/buildkite-experimental/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
```

Comment:
It's safe to remove all of the `buildkite-experimental` setup (better represented as tfvars as is the plan for the new terraform world).